README.md

```diff
@@ -19,3 +19,17 @@ I koden brukes prefikset `dds` eller `DDS` for å spesifisere assosiasjon med de
 ## 🤝 Bidra
 
 Team Elsa setter pris på all bidrag. Les hvordan bidra i [guiden for bidragsytere](https://design.domstol.no/987b33f71/p/34c962-hvordan-bidra/b/603442).
+
+### 🧑‍💻 Utvikling
+
+In£staller avhengigheter
```

Comment:
Typo eller kul slang? 😅

Reply:
Begge tenker jeg. Litt for kul slang til å ta med her dog😞